src/RequestExecutor/HttpRequestExecutor.ts

```diff
@@ -136,9 +141,16 @@ export class HttpRequestExecutor implements RequestExecutor {
       );
       logger.error('Cause: %s', message);
 
+      let statusCode: number = null;
+
+      if (err === HttpRequestExecutor.ERR_TIMEOUT) {
+        statusCode = HttpRequestExecutor.HTTP_STATUS_GATEWAY_TIMEOUT;
+      }
+
       return new Response({
         message,
         errorCode,
```

Comment:
Unify `errorCode` usage instead of introducing a new field. `statusCode` is semantically wrong, as error is not HTTP response.

Reply:
more context: the `ETIMEDOUT` error code is used everywhere in repeater for timeout errors. Instead of manually setting 502 status code, use the same error code.

Reply:
@derevnjuk done